api/admin/coordinator.py

```diff
@@ -53,8 +53,21 @@ class BaseCoordinatorGroupInline(
     max_num = 0
 
 
+class NameGroupChoiceField(forms.ModelChoiceField):
+
+    def label_from_instance(self, obj: Any) -> str:
+        coordinators = ",".join([str(c) for c in obj.coordinators.all()]).rstrip().strip()
+        return (
+            f"Group {obj.pk}, {obj.language_and_level} (coords: {obj.coordinators.count()}, "
```

Comment:
```suggestion
            f"Group {obj.pk}, {obj.language_and_level} ({obj.coordinators.count()} students, "
```
maybe you meant students?

Reply:
Wait, why students?

Reply:
Dunno, but you repeat coordinators twice, first their number, then their names, I just didn't think it was needed. OTOH, it might make sense to just show how many students are in the group.
Anyway, I may be confusing you needlessly

Reply:
yep, [I am](https://github.com/samanthasgroup/django-webapps/pull/483#discussion_r1552484422)

closing this thread